files/default/vendor/chef-server/fetcher.rb

```diff
@@ -50,11 +50,8 @@ def self.chef_server_org
     end
 
     def self.url_prefix
-      if chef_server_visibility?
-        ''
-      else
-        '/compliance'
-      end
+      return '/compliance' unless chef_server_visibility? && Chef.node.attributes['audit']['fetcher'] != 'chef-server'
```

Comment:
This method is getting harder to understand. 
I pinged Victoria as well and would like to keep it as if-then-else:

```ruby
def self.url_prefix
  if chef_server_visibility? && Chef.node.attributes['audit']['fetcher'] != 'chef-server'
    ''
  else
    '/compliance'
  end
end
```